pages/shelf/[[walletAddress]].vue

```diff
@@ -2,9 +2,35 @@
   <div>
     <AppHeader />
 
+    <header
+      v-if="isSomeoneElseBookshelf"
```

Comment:
```suggestion
      v-if="!isOwnedBookshelf"
```
I think flipped condition would be cleaner in naming, but just suggestion